goldstone/tests.py

```diff
@@ -166,12 +166,12 @@ def test_es_indices(self, m_conn):
         self.assertEqual(es_indices(), "_all")
 
         # test with no conn
-        result = es_indices(prefix='index')
+        result = es_indices(prefix='index1*')
         self.assertTrue(m_es.indices.status.called)
         self.assertIn('index1', result)
         self.assertNotIn('not_index1', result)
 
         # test with prefix
-        result = es_indices('index', es_conn())
+        result = es_indices('index1*', es_conn())
```

Comment:
would probably be better coverage to test a couple different conditions.  so, in this series of tests, we should have ones that test a string without a * a string with a *, and anything else that we might see in the real world.